swagauth/settings.py

```diff
@@ -112,19 +128,28 @@
 
 USE_TZ = True
 
-
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/2.2/howto/static-files/
 
 STATIC_URL = '/static/'
 
+SITE_ID = 1
+
+AUTHENTICATION_BACKENDS = [
+  # Needed to login by username in Django admin, regardless of `allauth`
+  'django.contrib.auth.backends.ModelBackend',
+
+  # `allauth` specific authentication methods, such as login by e-mail
+  'allauth.account.auth_backends.AuthenticationBackend',
+]
+
 SWAGAUTH_SETTINGS = {
-    'github': {
-        'SCOPE': [
-            'user',
-            'repo',
-            'read:org',
-        ],
+  'github': {
+    'APP': {
+      'client_id': 'b91d8237c69c8a8ff7af',
+      'secret': 'c94b9103f4a87398ecabd6ad715a98f7f318f44e',
```

Comment:
Don't store credentials in the project itself, use [django-environ](https://django-environ.readthedocs.io/en/latest/)

it should be something like that:
`'client': env('GUTHUB_CLIENT_ID'),`